templates/components/common/navigation-menu.html

```diff
@@ -2,231 +2,107 @@
-                            </a>
-                        </li>
-                        {{#if settings.returns_enabled}}
-                            <li>
-                                <a 
-                                    class="js-nav-pages-action"
-                                    href="{{urls.account.returns}}"
-                                    aria-label="{{lang 'account.nav.returns'}}"
+                                {{name}}
+                                <span 
+                                    data-collapsible="nav-pages-{{id}}"
+                                    data-collapsible-disabled-breakpoint="medium"
+                                    data-collapsible-disabled-state="open"
+                                    data-collapsible-enabled-state="closed"
                                 >
-                                    {{lang 'account.nav.returns'}}
-                                </a>
-                            </li>
-                        {{/if}}
-                        <li>
-                            <a 
-                                class="js-nav-pages-action"
-                                href="{{urls.account.inbox}}"
-                                aria-label="{{lang 'account.mobile_nav.messages'}}"
-                            >
-                                {{lang 'account.mobile_nav.messages'}}
-                            </a>
-                        </li>
-                        <li>
-                            <a 
-                                class="js-nav-pages-action"
-                                href="{{urls.account.addresses}}"
-                                aria-label="{{lang 'account.nav.addresses'}}"
-                            >
-                                {{lang 'account.nav.addresses'}}
+                                    <svg>
+                                        <use href="#icon-chevron-down" />
+                                    </svg>
+                                </span>
                             </a>
-                        </li>
-                        {{#if settings.show_payment_methods}}
-                            <li>
-                                <a 
-                                    class="js-nav-pages-action"
-                                    href="{{urls.account.payment_methods.all}}"
-                                    aria-label="{{lang 'account.nav.payment_methods'}}"
-                                >
-                                    {{lang 'account.nav.payment_methods'}}
-                                </a>
-                            </li>
+                            <ul id="nav-pages-{{id}}" class="is-open">
+                                {{#each children}}
+                                    <li>
+                                        <a 
+                                            class="js-nav-pages-action{{#if is_active}} is-active{{/if}} c-nav-page-action c-sub-menu-item"
```

Comment:
Mixing classes of different components on the same element - no so good for preventing styles overwriting